omod/src/main/java/org/openmrs/module/idgen/rest/resource/IdentifierSourceResource.java

```diff
@@ -85,16 +90,16 @@ public DelegatingResourceDescription getUpdatableProperties() throws ResourceDoe
         description.addProperty("description");
         return description;
     }
-
-    @PropertyGetter("identifierType")
-    public String getidentifierType(IdentifierSource identifierSource) {
-        return identifierSource.getIdentifierType().getUuid();
-    }
 	
     @PropertyGetter("display")
     public String getDisplayString(IdentifierSource identifierSource) {
         return identifierSource.getIdentifierType() + " - " + identifierSource.getName();
     }
+    
+    @PropertyGetter("sourceType")
```

Comment:
Dont think you need this property getter

Reply:
removed